src/config/configAggregator.ts

```diff
@@ -290,6 +290,17 @@ export class ConfigAggregator extends AsyncOptionalCreatable<JsonMap> {
     return this;
   }
 
+  /**
+   * Add an allowed config property.
+   */
+  public addAllowedProperties(configMetas: ConfigPropertyMeta | ConfigPropertyMeta[]): void {
+    if (isArray(configMetas)) {
```

Comment:
Why do we have an imported `isArray` method when we can just use `Array.isArray()`?

Reply:
I'm going to leave it for now since it is already brought in by ts-types, but your right we probably should be using that instead and just remove it from ts-types.